aten/src/ATen/core/dispatch/Dispatcher.h

```diff
@@ -173,66 +174,55 @@ inline Return Dispatcher::callUnboxed(const OperatorHandle& op, Args... args) co
   detail::unused_arg_(args...);  // workaround for a false-positive warning about unused parameters in gcc 5
 
   // note: this doesn't need the mutex because write operations on the list keep iterators intact.
-  return op.operatorIterator_->op.readDispatchTable([&] (const DispatchTable& dispatchTable) -> Return {
-    // TODO This should be a nested lambda instead of a separate function call, but that triggers an internal
-    // compiler error on GCC5. Change this once we don't need gcc 5 anymore.
-    return doCallUnboxed<Return, Args...>(dispatchTable, backendFallbackKernels_, std::forward<Args>(args)...);
-  });
+  // TODO This should be a nested lambda instead of a separate function call, but that triggers an internal
+  // compiler error on GCC5. Change this once we don't need gcc 5 anymore.
+  return doCallUnboxed<Return, Args...>(op.operatorIterator_->op.dispatch_table(), std::forward<Args>(args)...);
```

Comment:
Since this is not a lambda anymore, we can resolve the TODO and just inline `doCallUnboxed` here. Would make the code much easier. But doesn't necessarily have to happen in this PR.